hack/check-drenv-prereqs.sh

```diff
@@ -16,9 +23,23 @@ for cmd in "${commands[@]}"; do
   fi
 done
 
-if ! virt-host-validate qemu -q; then
-  echo "Error: 'virt-host-validate qemu' did not return 0."
-  exit 1
+
+if [[ "$os" == "Linux" ]]
+then
+  for cmd in "${linux_only_commands[@]}"; do
+    if ! command -v "$cmd" &> /dev/null; then
+      echo "Error: $cmd could not be found in PATH."
+      exit 1
+    fi
+  done
+fi
```

Comment:
This is unneeded duplication. We could do this instead:

    if linux:
        append virt-host-validate to commands

And use single loop for checking if commands exist.